frontend-typescript.js

```diff
@@ -9,6 +9,8 @@ const {
 const consistentTypeImports = require('./rules/consistent-type-imports.json');
 const frontendConfig = require('./frontend');
 const typescriptRules = require('./rules/typescript');
+const restrictedPackagesImportRules = require('./rules/restricted-packages-import.json');
+const youDontNeedLodashRules = require('./rules/you-dont-need-lodash.json');
 
 module.exports = {
   extends: ['./frontend.js', ...commonExtends, 'plugin:you-dont-need-lodash-underscore/compatible'],
```

Comment:
![CleanShot 2023-12-06 at 17 58 40@2x](https://github.com/shelfio/eslint-config/assets/33541009/30829a59-2c82-431d-bbe3-e6f303cfe4d5)

Not sure if this is needed. 
For frontend we are extinding compatible config, so its should already throw an error only on native methods

Reply:
it's throwing an error on omit and many other methods that are marked `compatible` in their config, there are many of those including `omit`
@slavik-chapelskyi